server/fast_forward.go

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package server
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/google/go-github/v39/github"
+	"github.com/mattermost/mattermost-mattermod/model"
+	"github.com/mattermost/mattermost-server/v5/mlog"
+)
+
+const (
+	cloudBranchName = "heads/cloud"
+	mainBranchName  = "heads/master"
+	processRepo     = "mattermost-server"
```

Comment:
Hmm .. curious to know about the name "processRepo"? I'd expect it to be validRepo or sourceRepo. 0/5 though. Just curious :)